oracle/price-feeder/oracle/provider/coinbase.go

```diff
@@ -150,7 +150,8 @@ func (p *CoinbaseProvider) GetTickerPrices(pairs ...types.CurrencyPair) (map[str
 	for _, currencyPair := range pairs {
 		price, err := p.getTickerPrice(currencyPair)
 		if err != nil {
-			return nil, err
+			p.logger.Warn().Msg(fmt.Sprint("failed to fetch tickers for pair ", currencyPair, " due to the following error ", err.Error()))
```

Comment:
Would this be too noisy? Shall we change to debug log as well since we have metrics now?

Reply:
yeah, we can change to debug log